variables.tf

```
@@ -72,11 +72,16 @@ variable "public_network_acl_id" {
 }
 
 variable "private_network_acl_id" {
-  default     = ""
   description = "Network ACL ID that will be added to private subnets. If empty, a new ACL will be created"
+  default     = ""
 }
 
 variable "nat_gateway_enabled" {
   description = "Flag to enable/disable NAT gateways for private subnets"
-  default     = "true"
+  default     = true
+}
+
+variable "map_public_ip_on_launch" {
+  default     = true
```

```suggestion
  default     = "true"
```

sorry, let's put it back to string.
let's make a local var and convert it to boolean there as we do with `nat_gateway_enabled` (for consistency)
Terraform recommends using strings instead of booleans in variables.
But at the same time many resources accept booleans.
That's why we usually introduce a local var to convert the string to a boolean and use it in the code.

And please rebuild README after that.
Thanks

---

@osulli please address the comment above

---

@aknysh - not converted bool to string before with local. Can you send example please?

---

```
locals {
  nat_gateways_count = "${var.nat_gateway_enabled == "true" ? length(var.availability_zones) : 0}"
}
```

 I see this but it doesn't mean too much to me.